src/routes/reference/kmeans.svx

```diff
@@ -27,3 +27,20 @@ In the widget below you can select from several different datasets. If you're on
 
 2. Inertia is not a normalised metric and exists on a scale of 0 to infinity. The only assumption we can make is that lower inertia values are better and 0 is ideal. In high-dimensional spaces, the [curse of dimensionality]() comes into play and the Euclidian distance values can become inflated, thus effecting the value of the inertia. If you are dealing with a high number of dimensions and [KMeans](/reference/kmeans) is not producing a useful clustering, then preprocessing the data with a dimension reduction algorithm such as [UMAP](/reference/umap), [MDS](/reference/mds) or [PCA](/reference/pca) can be valuable.
 
+## Spherical KMeans
+
+The FluCoMa toolset also provides another metric to compute distance to cluster. Instead of taking the Euclidian distance to each center of each cluster like [KMeans](/reference/kmeans) does, SKMeans takes the angular distance of the normalised vector in high dimension. This is also named cosine similarity. More information can be found [in this article](https://machinelearningcatalogue.com/algorithm/alg_spherical-k-means.html).
```

Comment:
> SKMeans takes the angular distance of the normalised vector in high dimension.

The high dimensionality isn't a given – it's still the dimensionality of the input data, same as kMeans. High dimensionality only comes into it, AFAIK because (a) cosine distance may fare better for clustering sparse and / or high dimensional input data and (b) when we use this object for feature learning, typically you'll pick a largeish *k* so that the data are projected into a higher dimensional space.